backend/app/store/service/service.go

```diff
@@ -996,3 +1049,13 @@ func (s *DataStore) getSecret(siteID string) (secret string, err error) {
 	}
 	return secret, nil
 }
+
+func (s *DataStore) indexComment(comment store.Comment) error {
```

Comment:
I think naming it `indexCommentForSearch` might be better for code readability.

Reply:
The reason is that it's outside the search module, and you need to look into the code of the function to know it's calling the search module.